test/vcf/metaentry_test.cpp

```diff
@@ -433,43 +433,322 @@ namespace ebi
             CHECK_NOTHROW( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "10"}, {"Type", "Integer"}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "10"}, {"Type", "Integer"}, {"Description", "Genotype"} },
                                 source
                             } ) );
                                 
             CHECK_NOTHROW( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "A"}, {"Type", "Float"}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "A"}, {"Type", "Float"}, {"Description", "Genotype"} },
                                 source
                             } ) );
                                 
             CHECK_NOTHROW( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "R"}, {"Type", "Character"}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "R"}, {"Type", "Character"}, {"Description", "Genotype"} },
                                 source
                             } ) );
                                 
             CHECK_NOTHROW( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "G"}, {"Type", "String"}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "G"}, {"Type", "String"}, {"Description", "Genotype"} },
                                 source
                             } ) );
                             
             CHECK_THROWS_AS( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "1"}, {"Type", "."}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "1"}, {"Type", "."}, {"Description", "Genotype"} },
                                 source
                             }),
                             vcf::MetaSectionError* );
                                 
             CHECK_THROWS_AS( (vcf::MetaEntry {
                                 1,
                                 "FORMAT",
-                                { {"ID", "GT"}, {"Number", "1"}, {"Type", "int"}, {"Description", "Genotype"} },
+                                { {"ID", "GH"}, {"Number", "1"}, {"Type", "int"}, {"Description", "Genotype"} },
+                                source
+                            }),
+                            vcf::MetaSectionError* );
+        }
```

Comment:
not sure about the utility of this function, now that GT has a defined Number "1"